labelbox/schema/project.py

```diff
@@ -177,6 +177,50 @@ def export_labels(self, timeout_seconds=60):
                          self.uid)
             time.sleep(sleep_time)
 
+    def export_issues(self, status=None, timeout_seconds=60):
+        """ Calls the server-side Issues exporting that generates a JSON
+        payload, and returns the URL to that payload.
+
+        Args:
+            timeout_seconds (float): Max waiting time, in seconds.
+            status (string): valid values: Open, Resolved
+        Returns:
+            URL of the data file with this Project's issues. If the server didn't
+            generate during the `timeout_seconds` period, None is returned.
+        """
+        sleep_time = 2
+        id_param = "projectId"
+        status_param = "status"
+        query_str = """query getProjectIssues($%s: ID!, $%s: IssueStatus) {
```

Comment:
Please change this to `getProjectIssuesPyApi` this naming convention is used on the backend to record metrics.

Reply:
Oh, I am reusing the same query that is used in product. Looks like I need to create a separete API just for SDK to make sure the convention is followed? This way I can be more specific about retries as well